Module.yaml

```diff
@@ -0,0 +1,74 @@
+version: 1.0.0
+name: OPCUAReader
+description: OPC UA Reader reads values for variables and forwards the payload to next module.
+icon: https://icons-020-demo.s3.eu-central-1.amazonaws.com/mi_OPCUA_input.png
```

Comment:
Again, please ask Christina for OPC UA process icon

Reply:
requested.

Reply:
module is changed to input based on our last call.